### DISCUSSION TITLE
Received Chinese data garbled

### DISCUSSION THREAD

**Discussion by ourlang** [nextgenhealthcare/connect#5960] [Frequently Asked Questions]
mirth connect Received Chinese data garbled  

charset encoding Already set UTF-8
![image](https://github.com/nextgenhealthcare/connect/assets/32355185/cc5724dd-3e6a-4ec6-a616-fb8da80f72eb)

java environment variable  Add parameters JAVA_TOOL_OPTIONS= -Dfile.encoding=UTF-8 -Duser.language=en -Duser.country=US

Received parameters or garbled code

![image](https://github.com/nextgenhealthcare/connect/assets/32355185/ea8696e2-391a-4860-a507-77add5e8e524)
![image](https://github.com/nextgenhealthcare/connect/assets/32355185/e911afa7-56f0-4fbe-a94f-d34d6c3751fa)

May I ask where else to set the encoding format for reception

**Comment by ourlang** [comment]
<img width="841" alt="1697696711204" src="https://github.com/nextgenhealthcare/connect/assets/32355185/23731854-5aee-43b6-923e-a2aa4a7bbd3e">

**Comment by jonbartels** [comment]
@yimengZJl  and @ahuiiiiiii have asked similar questions. Did they find a workaround?

Can you attach a CCD which contains Chinese characters as a file? This will help others reproduce the problem.

Is the message only broken when you view it in Mirth or is it also broken if export the message from Mirth?
